### PR TITLE
Added languages to unsorted

### PR DESCRIPTION
**Pull request by themak1985**
Did as many as I can - will try to get rest next time.

### CODE REVIEW

**Comment by themak1985** [comment]
@freearhey let me know if you would like this to be done differently

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@themak1985 Great job!

**Comment by themak1985** [comment]
@freearhey thanks for the invite - happy to help. Any processes you want me to follow? Chat elsewhere?

**Comment by freearhey** [comment]
@themak1985 Welcome!)

I don't have a lot of free time right now and it's getting hard to follow all the requests, so any help will come in handy.

**Comment by themak1985** [comment]
@freearhey got it, will be sure to tag you if I have any questions.
